### DISCUSSION TITLE
Publishing a release

### DISCUSSION THREAD

**Discussion by IntoxicatedCookie** [community/community#23349] [New to GitHub]
<p>Hello!</p>
<p>This is a silly question, but I wanted to be sure about this. Does publishing a release in my private Github repository, make it public? I only use Github to backup my code in case anything goes south, my code is not intended for public use and only I have access to it. I just wanted to publish a release to make it clear what the last stable release is.</p>
<p>I strongly suppose that it doesn’t make anything public, but I need to make sure it won’t.</p>
<p>Thanks in advance.</p>

**Comment by tajmone** [comment, marked as answer]
<aside class="quote no-group" data-username="LameProgrammer" data-post="1" data-topic="193666">
<div class="title">
<div class="quote-controls"></div>
 LameProgrammer:</div>
<blockquote>
<p>I strongly suppose that it doesn’t make anything public, but I need to make sure it won’t.</p>
</blockquote>
</aside>
<p>I don’t see why releases should affect the public status of a repository. Releases will be visible to whoever you have granted access to the repository, there’s no “public release” implied, a release is just a tag creating a meaningful snapshot of a version number (or whatever) to which associate release notes, and for which GH offers Zip archives of the source for download, and you can attach your own binaries too.</p>

**Comment by IntoxicatedCookie** [comment]
<p>I didn’t really see a point on why this should affect the visibility either, but I really needed to double check. This covers all the information I need. Thanks a lot for this! Sorry for the silly question…</p>
